src/backend/src/utils/reimbursement-requests.utils.ts

```diff
@@ -201,7 +201,7 @@ const createNewProducts = async (products: ReimbursementProductCreateArgs[], rei
   }
 };
 
-export type UserWithTeam = User & { teams: Team[] };
+export type UserWithTeam = User & { teamsAsMember: Team[] };
```

Comment:
ugh im now realizing were gonna have to change all the checks to also include the teams leads on top of the members